### DISCUSSION TITLE
Fav sources

### DISCUSSION THREAD

**Discussion by abaracedo** [dailydotdev/daily#922] [Feature Requests]
It would be a nice feature to have the option to mark sources as favs and list them under a new section. Sometimes I don't know if a source that I like to read has published a new post as my feed displays lots of content of my interest.

**Comment by idoshamun** [comment]
Hi @abaracedo,

Thanks for the suggestion.
In the past, we've considered letting users subscribe to a source and get a notification whenever there's a new post.
Would that work for you?

**Reply by abaracedo** [reply]
Sure, it's similar like marking as fav a source.
